src/domain/models/create-document-revision-body-schema.spec.js

```diff
@@ -85,9 +86,9 @@ describe('create-document-revision-request', () => {
       describe(description, () => {
         it(expectToPass ? 'should pass' : 'should fail', () => {
           if (expectToPass) {
```

Comment:
Actually we expect all tests to pass, so `expectToPass` is a bit misleading.
What about separating test cases into `positiveCases` and `negativeCases`?
This would remove the `if` condition in the assertion, would also be nicer.